velox/docs/functions/spark/datetime.rst

```diff
@@ -129,7 +129,7 @@ These functions support TIMESTAMP and DATE input types.
 
     Returns the date from year, month and day fields.
     ``year``, ``month`` and ``day`` must be ``INTEGER``.
-    Throws an error if inputs are not valid.
+    Returns NULL if inputs are not valid. (ANSI mode is off by default.)
```

Comment:
Perhaps we don't need to mention the 'ANSI mode' as it is for Spark while Velox does not contain such configuration or mode now.